swiss-ng-api.yaml

```diff
@@ -6137,10 +6138,12 @@ components:
 #        instructionIdentification: # N.A. for all, but can be used by ASPSP if needed
 #          type: string
 #          maxLength: 35
-#        debtorName: # N.A. for all, but can be used by ASPSP if needed
-#          $ref: "#/components/schemas/debtorName"
-#        debtorId: # N.A. for all, but can be used by ASPSP if needed
-#          $ref: "#/components/schemas/debtorId"
+        debtorName: # N.A. for all, but can be used by ASPSP if needed
+          $ref: "#/components/schemas/debtorName"
+        debtorId: # N.A. for all, but can be used by ASPSP if needed
+          $ref: "#/components/schemas/debtorId"
+        debtorAgent:
+          $ref: "#/components/schemas/deptorAgent7-CH"
```

Comment:
Just so I understand this correctly:  All single payments in a bulk payment are paid from the same account (`debtorAccout`). But each payment in a bulk payment can specify a different `debtorName` (e.g. _Hans Huber_, _H. Huber_ und _Mickey Mouse_) although the `debtorAccount` is the same? I can understand that the display name may vary per payment. But what about `debtorId`? Is it really possible to have different 'debtorId' although the `debtorAccount` is the same? Same problem with the `debtorAgent`

Reply:
I can understand your reasoning, we are following the Berlin Group definitions accurately here (see screenshot).
However, there are two things to consider. 

1. `debtorName` and `debtorID` are labeld "n.a." in Berlin Group's standard specification. My suggestion: no deviation from the standard and therefore no change
2. `debtorAgent` is a Swiss specific data element. My suggestion: handling it like `debtorAccount`

![grafik](https://user-images.githubusercontent.com/51756488/122243686-9bef5f80-cec4-11eb-9b3f-b2c6fbd002bb.png)

Reply:
> `debtorAgent` is a Swiss specific data element. My suggestion: handling it like `debtorAccount`

You mean, place it on the level of the `debtorAccount` and not in `payments`?

Reply:
Yes.